ebcli/controllers/create.py

```diff
@@ -564,6 +564,12 @@ def check_elb_type_from_configs(use_saved_config=False):
                     try:
                         config = yaml.safe_load(f)
                         option_settings = config.get('option_settings', [])
+                        if isinstance(option_settings, dict):
```

Comment:
This does address the issue, but we should also add the change on line 552, depending on the syntax, yes?